packages/gestalt/src/utils/omit.js

```diff
@@ -0,0 +1,16 @@
+// @flow strict
+
+const contains = (key, arr) => arr.indexOf(key) >= 0;
+// $FlowExpectedError[unclear-type]
+const omit = (keys: Array<string>, obj: Object): Object =>
```

Comment:
we can't know anything about the object used here, so I don't see a way around this suppression